src/App.js

```diff
@@ -44,15 +54,32 @@ export default function App() {
         <Route exact path="/rsvp-form" element={<RsvpForm />} />
         <Route path="/event/:event_id" element={<Frame />}>
           <Route path="dashboard" element={<Dashboard />} />
-          <Route path="rsvp-preview" element={<RSVP />} />
-          <Route path="invite" element={<InvitationForm />} />
           <Route path="registry" element={<Registry />} />
           <Route path="registrystart" element={<RegistryStart />} />
           <Route path="registryadd" element={<RegistryAdd />} />
           <Route path="browse-theme" element={<BrowseTheme />} />
-          <Route path="choose-content" element={<ChooseContent />} />
-          <Route path="invitation-form" element={<InvitationForm />} />
-          <Route path="rsvp-preview" element={<RSVP />} />
+          <Route
+            path="choose-content"
+            element={<ChooseContent setChosedTheme={setChosedTheme} />}
+          />
+          <Route
+            path="invitation-form"
+            element={
+              <InvitationForm
```

Comment:
related to the above comment, but would we need to pass the data down from this level?